### PR TITLE
Merge feature/102-alias

### LINKED ISSUE #102: Add vhost:alias command
Add a `vhost:alias` command to add a new alias to the `ServerAlias` line in the site vhost.

### PR DESCRIPTION
**Pull request by stevenrombauts**
Closes #102

### CODE REVIEW

**Comment by yiendos** [comment]
@stevenrombauts working like a 'ream (dream) for apache stuff, seems nginx relies on `server_name` could we work in support for nginx as well? Then we could most definitely hook up the ngrok share process to `vhost:alias`.

**Comment by stevenrombauts** [comment]
@yiendos Good point, forgot about that! I've added Nginx support, could you give that another test please?

File: src/Joomlatools/Console/Command/Vhost/Alias.php [hunk 1]
```diff
@@ -0,0 +1,136 @@
...
+        $site   = $this->site;
+        $alias  = $input->getArgument('alias');
+        $delete = $input->getOption('delete');
+
+        $changed = $this->_updateAliases($site, $alias, $delete, 'apache');
+        $changed = $this->_updateAliases($site, $alias, $delete, 'nginx') || $changed;
+
+        if ($changed && Util::isJoomlatoolsBox())
+        {
+            `box server:restart apache nginx`;
```
Comment (yiendos) [inline src/Joomlatools/Console/Command/Vhost/Alias.php]: @stevenrombauts just following this terminal command:
https://github.com/joomlatools/joomlatools-vagrant/blob/master/puppet/environments/box/modules/profiles/files/box/cli/scripts/Command/Server/Restart.php#L50

It thought we were supposed to be using `sudo systemctl restart apache2` from now on?

**Comment by yiendos** [comment]
@stevenrombauts looking good to me sir, code makes sense and works great.

File: src/Joomlatools/Console/Command/Vhost/Alias.php [hunk 2]
```diff
@@ -98,7 +98,7 @@ protected function _updateAliases($site, $alias, $delete = false, $application)
 
                 $parts = array_filter($parts);
                 $key   = array_search($alias, $parts);
-;
+
```
Comment (stevenrombauts) [inline src/Joomlatools/Console/Command/Vhost/Alias.php]: @yiendos Whoops  🤭 Glad you caught that one!

**Pull request merged**